doc/OVERMAP.md

```diff
@@ -247,7 +247,7 @@ rotation for the referenced overmap terrains (e.g. the `_north` version for all)
 | `color`           | Color to draw the symbol in. See [COLOR.md](COLOR.md).                                           |
 | `looks_like`      | Id of another overmap terrain to be used for the graphical tile, if this doesn't have one.       |
 | `vision_levels`   | Id of a `oter_vision` that describes how this overmap terrain will be displayed when there is not full vision of the tile.
-| `connect_group`   | Specify that this overmap terrain might be graphically connected to its neighbours, should a tileset wish to.  It will connect to any other `overmap_terrain` with the same `connect_group`. |
+| `connect_group`   | Specify that this overmap terrain might be graphically connected to its neighbors, should a tileset wish to.  It will connect to any other `overmap_terrain` with the same `connect_group`. |
```

Comment:
```suggestion
| `connect_group`   | Specify that this overmap terrain might be graphically connected to its neighbours, should a tileset wish to.  It will connect to any other `overmap_terrain` with the same `connect_group`. |
```